airbyte-integrations/connectors/source-github/source_github/schemas/teams.json

```diff
@@ -5,9 +5,6 @@
     "organization": {
       "type": ["null", "string"]
     },
-    "repository": {
```

Comment:
Is removing a field a breaking change?

Reply:
According to CAT tests removing field - is not breaking change
I just checked, I have removed one field and run CAT tests - no warnings about breaking changes